seed/context.py

```diff
@@ -0,0 +1,48 @@
+import collections
+import json
+import pathlib
+from typing import Any
+
+from mpt_api_client.models import Model
+
+
+class Context(collections.UserDict[str, Any]):
+    """Application context."""
+
+    def __init__(self, context_file: pathlib.Path | None = None) -> None:
+        super().__init__()
+
+    def load(self, context_file: pathlib.Path) -> None:
```

Comment:
It is a good practice always to separate configuration and the way how do you load the configuration from anywhere (file or api) :-)

So for loading I would prefer to have separate standalone function